tests/integration/targets/rds_cluster/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- module_defaults:
+    group/aws:
+      region: "{{ aws_region }}"
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+  collections:
+    - community.aws
+    - amazon.aws
+
+  block:
+  - include_tasks: create_cluster.yml
+  
+  # Failing
+  # - include_tasks: restore_cluster.yml
+
+  # TODO: An error occurred (InvalidParameterValue) when calling the CreateDBCluster operation: Replication from cluster in same region is not supported
+  # - include_tasks: promote_cluster.yml
+
+  - include_tasks: create_cluster_vpc_sgs.yml
+  - include_tasks: modify_cluster.yml
+  - include_tasks: tag_cluster.yml
```

Comment:
So if these tests take anything as long as the `rds_instance` ones then it might be worth running batches in parallel like done here, which even if the tests finish within the 1hr max session then it could speed them up as much as possible being the slow updating service it is:
https://github.com/ansible-collections/community.aws/tree/main/tests/integration/targets/rds_instance

Reply:
@marknet15 Thank you, you're right. Will try to replicate.